src/convertToObject.js

```diff
@@ -3,16 +3,44 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split('\n');
+  const trimmed = [];
+
+  stylesArray.forEach((style) => {
+    trimmed.push(style.trim().replace(';', ''));
+  });
+
+  const cleared = [];
+
+  trimmed.forEach((e) => {
+    if (e) {
+      cleared.push(e.split(':'));
```

Comment:
Instead of checking the truthiness of 'e', it would be better to specifically check if 'e' is not an empty string. This will avoid skipping properties which have falsy values like 0.